crates/evm/src/tests/queries/estimate_gas_tests.rs

```diff
@@ -93,7 +93,7 @@ fn test_tx_request_fields_gas() {
     );
     assert_eq!(
         contract_diff_size.unwrap(),
-        serde_json::from_value::<EstimatedDiffSize>(json![{"gas":"0x6601","diffSize":"0xa8"}])
+        serde_json::from_value::<EstimatedDiffSize>(json![{"gas":"0x6601","diffSize":"0xdc"}])
```

Comment:
here for example, call was made with non eip1550 tx request and the previous diff size calculation failed to add coinbase (addr. and balance change, 20 + 32)